### PR TITLE
HTS-809: Selenium tests added

### PR DESCRIPTION
**Pull request by sureshhmrc**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -2,18 +2,12 @@ import play.core.PlayVersion
 import sbt.Keys._
 import sbt._
 import uk.gov.hmrc.DefaultBuildSettings._
-import uk.gov.hmrc.{SbtAutoBuildPlugin, _}
 import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin
 import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin._
 import uk.gov.hmrc.versioning.SbtGitVersioning
+import uk.gov.hmrc.{SbtAutoBuildPlugin, _}
```
Comment (sureshhmrc) [inline build.sbt]: All the selenium scripts and code are copied from front-end  -  the only new bits are `pages` and `features` packages

Comment (sureshhmrc) [reply build.sbt]: Also the `/run_selenium_system_test_local_chrome_service_manager.sh` script is not copied from frontend as the stride stubs are not running via service manager

**Comment by sureshhmrc** [comment]
All the feature tests are running fine but locally seeing an issue which i am looking at the moment
```Sureshs-MacBook-Pro:help-to-save-stride-frontend suresh$ ./run_selenium_system_test.sh 
JAVA options are: 
[warn] Executing in batch mode.
[warn]   For better performance, hit [ENTER] to switch to interactive mode, or
[warn]   consider launching sbt without any commands, or explicitly passing 'shell'
[info] Loading global plugins from /Users/suresh/.sbt/0.13/plugins
[info] Loading project definition from /Users/suresh/work/help-to-save-stride-frontend/project
[info] SbtAutoBuildPlugin - LICENSE file exists, sbt-header will add Apache 2.0 license headers to each source file.
[info] SbtAutoBuildPlugin - adding 36 build settings
[info] sbt git versioned as 0.15.0-3-gf1d3106
[info] sbt git versioned as 0.15.0-3-gf1d3106
[info] Set current project to help-to-save-stride-frontend (in build file:/Users/suresh/work/help-to-save-stride-frontend/)
<set>:1: error: not found: value SeleniumTest
javaOptions in SeleniumTest ++= Seq()
               ^
[error] Type error in expression
Sureshs-MacBook-Pro:help-to-save-stride-frontend suresh$ 
```

File: selenium-system-test/src/test/scala/htsstride/driver/Driver.scala [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    val options = new ChromeOptions()
+    options.addArguments("test-type")
+    options.addArguments("--proxy-server=http://localhost:11000")
+    capabilities.setCapability(ChromeOptions.CAPABILITY, options)
+    val driver = new ChromeDriver(capabilities)
+    val caps = driver.getCapabilities
+    driver
+  }
+
+  def createBrowserStackDriver: WebDriver = useBrowserStackDriverConstructor("test", default = true)
```
Comment (AndyHWChung) [inline selenium-system-test/src/test/scala/htsstride/driver/Driver.scala]: do we need the browserstack stuff here?

File: selenium-system-test/src/test/resources/browserstackdata/BS_Android_GalaxyS5.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
```
[the remark this answers is not in the document]
Comment (sureshhmrc) [reply selenium-system-test/src/test/resources/browserstackdata/BS_Android_GalaxyS5.json]: not sure if this is needs by selenium in the background- i will check with  Marcus

[the remark this answers is not in the document]
Comment (AndyHWChung) [reply selenium-system-test/src/test/resources/browserstackdata/BS_Android_GalaxyS5.json]: you can probably check with @Jackie1988 as well - she was the one who implemented all the browserstack stuff before

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**